File: LTC2.Shared.Models/Domain/Track.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace LTC2.Shared.Models.Domain
{
    public class Track
    {
        public Track()
        {
        }

        public Track(Visit visit)
        {
            ExternalId = visit.ExternalId;
            Name = visit.Name;
            Coordinates = visit.Track;
            Distance = visit.Distance;
            VisitedOn = visit.VisitedOn;
            Updated = visit.Updated;
            Places = visit.VisitedPlaces;
        }

        public DateTime VisitedOn { get; set; }

        public string Name { get; set; }

        public long Distance { get; set; }

        public string ExternalId { get; set; }

        public List<List<double>> Coordinates { get; set; }

        public List<string> Places { get; set; }

        [JsonIgnore]
        public bool Updated { get; set; }
    }
}

File: LTC2.Webclients.MainClient/ltc2-main/src/models/Track.ts
import { emptyString } from './Constants'

export class Track {
    public externalId: string = emptyString;
    public name: string = emptyString;
    public coordinates: number[][] = [];
    public distance: number = 0;
    public visitedOn: string = emptyString;
    public places: string[] = [];
}